manifests/rule/remote.pp

```diff
@@ -160,7 +166,7 @@
   Optional[String]                                 $stream_driver                        = undef,
   Integer[0]                                       $stream_driver_mode                   = 1,
   String                                           $stream_driver_auth_mode              = 'x509/name',
-  String                                           $stream_driver_permitted_peers        = "*.${::domain}",
+  String                                           $stream_driver_permitted_peers        = 'LOGSERVERNAME',
```

Comment:
Why not have it optional with a default of undef and note in the parameter description that it will be set to the log server if not set?

Reply:
So I got rid of the LOGSERVERNAME stuff.  I used an Array, even though you can use a comma seperate list of names in a string and just joined the array into a string.

I killed ny rsyslog changes because they were not needed any more.